webviz_subsurface/plugins/_inplace_volumes_onebyone.py

```diff
@@ -340,15 +339,17 @@ def filter_selectors(self):
                         open=True,
                         children=[
                             html.Summary(selector.lower().capitalize()),
-                            dcc.Dropdown(
+                            wcc.Select(
                                 id=self.selectors_id[selector],
                                 options=[
                                     {"label": i, "value": i}
                                     for i in list(self.volumes[selector].unique())
                                 ],
                                 value=list(self.volumes[selector].unique()),
                                 multi=True,
-                                clearable=False,
+                                size=20
+                                if len(list(self.volumes[selector].unique())) > 20
+                                else len(list(self.volumes[selector].unique())),
```

Comment:
```suggestion
                                size=min(20, len(list(self.volumes[selector].unique()))),
```

Reply:
Does btw. `len(self.volumes[selector].unique())` work, without casting to `list`?